lblod/frontend-lokaal-mandatenbeheer
# add other mandates when selecting mayor too

## Description

add other mandates when selecting mayor too. Note: this requires a page refresh to show the new mandataris instances if  no such role exists, but since we will be prefilling the mandates with empty instances, this should not be an issue.

## How to test

Change the burgemeester. See the mandataris for the voorzitter vast bureau and voorzitter bcsd update too.
When adding multiple mandataris instances with those roles only the first one will be updated and a warning will be shown.

![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/1076194/2a1422d3-de83-4b3f-a4d5-40a0a4b1289f)
